packages/client/hmi-client/src/components/operator/tera-operator-port-icon.vue

```diff
@@ -14,3 +14,8 @@ const icons = new Map<string, string>([
 	['modelConfigId', 'pi-cog']
 ]);
 </script>
+<style scoped>
+.p-chip-icon {
+	font-size: var(--font-body);
```

Comment:
I don't think this `--font-body` variable exist? where is it defined?
Would it be "--font-body-small", "--font-body-large"?